docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-media-player-data-model/index.md

```diff
@@ -17,125 +17,64 @@ import Badges from '@site/src/components/Badges';
+  - The media base module outputs the `snowplow_media_player_base` table.
+  - It also produces a `_pivot_base` table to calculate the percent_progress boundaries and weights that are used to calculate the total play_time and other related media fields.
+- **Media plays:** Removes impressions from the media base table to summarize media plays on a page_view by media_id level.
+  - The media plays module outputs the `snowplow_media_player_plays_by_pageview` view.
+- **Media stats:** Aggregates the media base table to individual `media_id` level, calculating the main KPIs and overall video/audio metrics.
+  - It uses the native dbt incremental materialization on a pageview basis after a set time window passed. This is to prevent complex and expensive queries due to metrics which need to take the whole page_view events into calculation. This way the metrics will only be calculated once per pageview / media, after no new events are expected.
+  - The media stats module outputs the `snowplow_media_player_media_stats` table.
+- **Media ad views:** Summarizes metadata and KPIs for each ad view within a media playback. Each ad a user viewed will result in a single row with information about the progress reached, whether the ad was skipped or clicked.
+  - The media ad views module outputs the `snowplow_media_player_media_ad_views` table.
+- **Media ads:** Aggregates all ad views to produce a summary of the KPIs for each ad played within each media content. Produces total counts of views, skips, clicks and progress reached. It also counts the metrics in terms of unique users (e.g., number of users who clicked the ad).
+  - It outputs the `snowplow_media_player_media_ads` table.
 
-</TabItem>
+## Mixing web and mobile events
 
-</Tabs>
+The package makes no distinction between events tracked from the web and those tracked from a mobile application, so long as you are tracking media events and from allowed `app_id`s. The `sessionId` from the `client_session` context, and the `id` from the `mobile_screen` context, overwrite the `domain_sessionid` and `page_view_id` fields respectively in our intermediate and derived tables, for events where they are populated. If you are just using web events, the package will work out the box. If you are using a mix of web and mobile events, you will need to enable set the `snowplow__enable_mobile_events` package variable to `true` and events will be processed from both sources. If you are only tracking mobile events, the package will work with the variable set, however you must have the [`webPage` context](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracker-setup/initialization-options/index.md#adding-predefined-contexts) available in your warehouse (even though it will not be populated for these mobile events) for the package to run.
```

Comment:
```suggestion
The package makes no distinction between events tracked from the web and those tracked from a mobile application, so long as you are tracking media events and from allowed `app_id`s. The `sessionId` from the `client_session` context, and the `id` from the `mobile_screen` context, overwrite the `domain_sessionid` and `page_view_id` fields respectively in our intermediate and derived tables, for events where they are populated. If you are just using web events, the package will work out the box. If you are using a mix of web and mobile events, you will need to set the `snowplow__enable_mobile_events` package variable to `true` and events will be processed from both sources. If you are only tracking mobile events, the package will work with the variable set, however you must have the [`webPage` context](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracker-setup/initialization-options/index.md#adding-predefined-contexts) available in your warehouse (even though it will not be populated for these mobile events) for the package to run.
```

Reply:
removing `enable` and leaving set the `snowplow__enable_mobile_events` package variable to `true`